From d2afffccb99ec989fc94e05b68b2858dadd66128 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Niklas=20S=C3=B6derlund?= <niso@kth.se>
Date: Thu, 11 Aug 2011 23:50:19 +0200
Subject: [PATCH] Add missing function

---
 file.c |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/file.c b/file.c
index dd39f5f..c603b59 100644
--- a/file.c
+++ b/file.c
@@ -73,6 +73,11 @@ static size_t __pram_iov_copy_to(char *vaddr,
 	return copied - left;
 }
 
+static inline int unsigned_offsets(struct file *file)
+{
+	return file->f_mode & FMODE_UNSIGNED_OFFSET;
+}
+
 static size_t pram_iov_copy_from(void *to, struct iov_iter *i, size_t bytes)
 {
 	size_t copied;
-- 
1.7.6

